.travis.yml

```diff
@@ -9,15 +9,32 @@ python:
   - "3.5"
   - "pypy"
   - "pypy3"
+matrix:
+  include:
+    - python: "2.7"
+      env: TOXENV=perf
 
-install:
-  - pip install -e .
-script:
-  - py.test
+before_install: |
+  pip install tox "virtualenv<14.0.0" # recent versions don't run on Python 3.2
```

Comment:
Sounds reasonable to me. Although, I'd rather drop it in a separate PR, where we could get rid of a workaround from #31 too (I mean `u(...)` stuff).

BTW do you know, are there any plans to drop 3.2 in pytest-core?